notebooks/Emukit-tutorial-bayesian-optimization-integrating-model-hyperparameters.ipynb

```diff
@@ -0,0 +1,385 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Bayesian optimization integrating model hyper-parameters"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this notebook we are going to see how to use Emukit to solve optimization problems the acquisition function is integrated with respect to the hyper-parameters of the model. \n",
```

Comment:
just some typos:

word missing: ... solve optimization problems __when__ the acquisition function is...

Reply:
Corrected